src/super_gradients/training/transforms/transforms.py

```diff
@@ -777,66 +800,30 @@ def __init__(
         self.min_bbox_edge_size = min_bbox_edge_size
         self.max_targets = max_targets
 
-    def __call__(self, sample):
-        normalized_input = "NORMALIZED" in self.input_format.value
-        normalized_output = "NORMALIZED" in self.output_format.value
-        normalize = not normalized_input and normalized_output
-        denormalize = normalized_input and not normalized_output
-
-        label_first_in_input = self.input_format.value.split("_")[0] == "LABEL"
-        label_first_in_output = self.output_format.value.split("_")[0] == "LABEL"
-
-        input_xyxy_format = "XYXY" in self.input_format.value
-        output_xyxy_format = "XYXY" in self.output_format.value
-        convert2xyxy = not input_xyxy_format and output_xyxy_format
-        convert2cxcy = input_xyxy_format and not output_xyxy_format
-
-        image, targets, crowd_targets = sample["image"], sample["target"], sample.get("crowd_target")
+        self.convert_format = DetectionFormatAdapter(input_format=self.input_format, output_format=self.output_format, image_shape=image_shape)
```

Comment:
Name is misleading (even if it looks nice in call()). Please rename.

Reply:
Renamed like that: https://github.com/Deci-AI/super-gradients/blob/0f65b88cdfba6ceb791bb0927ea28d87859e855d/src/super_gradients/training/transforms/transforms.py#L787